### PR TITLE
podman build logic

### PR DESCRIPTION
**Pull request by dofinn**

### CODE REVIEW

File: boilerplate/_lib/container-make [hunk 1]
```diff
@@ -12,14 +12,21 @@ source ${0%/*}/common.sh
 CONTAINER_ENGINE=$(command -v podman || command -v docker)
 [[ -n "$CONTAINER_ENGINE" ]] || err "Couldn't find a container engine. Are you already in a container?"
 
+CONTAINER_ENGINE_SHORT=$(echo $CONTAINER_ENGINE | awk -F '/' '{print $NF}')
```
Comment (2uasimojo) [inline boilerplate/_lib/container-make:15]: Or moar shellier

```suggestion
CONTAINER_ENGINE_SHORT=${CONTAINER_ENGINE##*/}
```

**Review by 2uasimojo — APPROVED** [review]
This wfm. If it wfy, I say let's do it!

/lgtm

/hold

@jharrington22 please validate in your env, unhold if ya like it.

**Comment by ArielLima** [comment]
@2uasimojo Just out of curiosity, do you guys have a plan on merging this soon? I am able to use boilerplate with it

**Comment by 2uasimojo** [comment]
Sorry, we were waiting for @jharrington22  to validate, since he had similar issues and had proposed an alternative solution. But I’m happy to take your vote instead :)

/hold cancel

**Pull request merged**